### DISCUSSION TITLE
Monitorear celular

### DISCUSSION THREAD

**Discussion by ysernah** [eusonlito/GPS-Tracker#58] [Q&A]
Hola amigo pregunta habra algun manual de como configurar para poder monitorear un celular android con traccar app ya que he probado sin exito

**Comment by eusonlito** [comment]
Has probado según la documentación de la Wiki? https://github.com/eusonlito/GPS-Tracker/wiki/%5BES%5D-Aplicaciones-M%C3%B3vil

**Reply by ysernah** [reply]
No ve revisado gracias por el link lo revisare

**Reply by ysernah** [reply]
Hola amigo lo he probado y tengo error de app traccar fallo de envio
Mi servidor es 192.168.1.12:8000 y es accesible desde la red local y esa es la direccion que configuro en traccar.

**Reply by eusonlito** [reply]
No puedes conectar el dispositivo usando tu red local ya que el dispositivo al usar la conexión mediante tarjeta de datos necesitas disponer de un ip pública, o bien configurar tu router para redirigir el tráfico del puerto 8000 a tu ordenador, siempre y cuando tengas internet con IP fija. Lo ideal es siempre realizar una instalación en un servidor normal.

**Reply by ysernah** [reply]
Aya amigo con razon habra algun manual para instalar en un hosting o vps?

**Reply by eusonlito** [reply]
Lo siento, pero ahí ya no puedo ayudarte 😅 necesitarías a alguien con conocimientos de sistemas para que te asista.
